jerry-core/parser/regexp/re-parser.cpp

```diff
@@ -271,6 +277,39 @@ re_count_num_of_groups (re_parser_ctx_t *parser_ctx_p) /**< RegExp parser contex
 } /* re_count_num_of_groups */
 
 /**
+ * Helper function for \x and \u. Read hexa or decimal numbers and convert to
+ * unicode codeunit.
+ *
+ * @return completion value
+ *         Returned value must be freed with ecma_free_completion_value
+ */
+static ecma_completion_value_t
+re_get_codeunit_from_string (lit_utf8_iterator_t *iter, /**< input string iterator*/
+                             ecma_char_t *codeunit, /**< Output: read codeunit */
```

Comment:
Please, add `_p` suffixes to pointer variables, and `out_` prefixes to output arguments.